app/src/main/java/ch/epfl/sweng/eventmanager/repository/data/Event.java

```diff
@@ -57,4 +66,24 @@ public EventOrganizer getOrganizer() {
     public Bitmap getImage() {
         return image;
     }
+
+    public List<Concert> getConcertList() {
+        return concertList;
+    }
+
+    /**
+     * Adds a concert to the event's concert list.
+     * The method will return true if the concert already exists in the list.
+     * @param concert Concert to be added to the List, can be null.
+     * @return true if the concert is in the list after returning.
+     */
+    public boolean addConcert(Concert concert) {
```

Comment:
The idea of this class is to be imutable-ish, adding concerts breaks this goal. Remove this method.

Reply:
I implemented it as a reminder that the EventOrganizer can modify his event's schedule.

Reply:
We don't really know how we will do that